docs/software/sciapps/cp2k.md

```diff
@@ -408,6 +413,45 @@ See [manual.cp2k.org/CMake] for more details.
 
 ### Known issues
```

Comment:
Sorry, one more thing I just noticed... This was already the case, but "Known issues" is a subheading under "Building CP2K from source". That doesn't seem quite right. Should the heading (and all its subheadings) be bumped one level up?

Reply:
Yes, good catch, thanks.

Reply:
Fixed, I also corrected a few typos.